src/main/java/com/nvs/store/controllers/ProductController.java

```diff
@@ -30,6 +28,10 @@ public ResponseEntity<Product> getProduct(@PathVariable Long id) {
     public ResponseEntity<Product> addProduct(@RequestBody Product product) {
         return productService.addProduct(product);
     }
+    @PutMapping("{id}")
+    public ResponseEntity<Product> updateProduct (@PathVariable Long id , @RequestBody Product product){
```

Comment:
Please use the formatting functionality from IDE to get rid of unnecessary spaces, and in general to format the source code files.

Reply:
done